test/issue1416-maven-repo-pkg-supplier.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+DIR=$(dirname "${BASH_SOURCE[0]}")
+
+. "$DIR"/common.sh
+
+PORT=$(($$ + 1024)) # PID + 1024
+
+dub remove maven-dubpackage --non-interactive --version=* 2>/dev/null || true
+
+"$DUB" build --single "$DIR"/test_maven_registry.d
+"$DIR"/test_maven_registry --folder="$DIR/issue1416-maven-repo-pkg-supplier" --port=$PORT &
+PID=$!
+sleep 1
+trap 'kill $PID 2>/dev/null || true' exit
+
+echo "Trying to download maven-dubpackage (1.0.5)"
+"$DUB" fetch maven-dubpackage --version=1.0.5 --skip-registry=all --registry=mvn+http://localhost:$PORT/maven/release/dubpackages
+
+if ! dub remove maven-dubpackage --non-interactive --version=1.0.5 2>/dev/null; then
+    die 'DUB not have installed package from maven registry.'
```

Comment:
Spelling:

- DUB did not install
- DUB doesn't have the installed

Reply:
done